src/main/java/uk/gov/hmcts/divorce/solicitor/event/page/SolAboutApplicant2.java

```diff
@@ -29,7 +29,13 @@ public void addTo(final PageBuilder pageBuilder) {
             .complex(CaseData::getApplicant2)
                 .mandatoryWithLabel(Applicant::getFirstName,
                     "${labelContentTheApplicant2UC}'s first name")
-                .mandatoryWithLabel(Applicant::getMiddleName,
+                .mandatory(Applicant::getMiddleName,
```

Comment:
I think there is some kind of misunderstanding - this should always be optional regardless of joint or sole application